stdlib/builtins.pyi

```diff
@@ -1547,6 +1547,7 @@ def __import__(
     fromlist: Sequence[str] = ...,
     level: int = ...,
 ) -> types.ModuleType: ...
+def __build_class__(__func: Callable[[], Any], __name: str, *bases: type, metaclass: type = ..., **kwds: Any) -> type: ...
```

Comment:
```suggestion
def __build_class__(__func: Callable[[], Any], __name: str, *bases: Any, metaclass: Any = ..., **kwds: Any) -> Any: ...
```
While conventionally these are all types, there is no strict requirement that they are. For example, the bases could just be objects providing `__mro_entries__`. We could be more precise here, but any code using `__build_class__` directly probably isn't going to make type checkers very happy anyway.